api/all/src/main/java/io/opentelemetry/api/metrics/DoubleGaugeBuilder.java

```diff
@@ -65,4 +65,17 @@ public interface DoubleGaugeBuilder {
   default ObservableDoubleMeasurement buildObserver() {
     return DefaultMeter.getInstance().gaugeBuilder("noop").buildObserver();
   }
+
+  /**
+   * Builds and returns a DoubleGauge instrument with the configuration.
+   *
+   * <p>NOTE: This produces a synchronous gauge which records gauge values as they occur. Most users
```

Comment:
Maybe something about how only the most recent one is exported, since I think that part of synchronous gauges is most likely to trip people up?

Reply:
I'll add a comment to that effect.